core-java-modules/core-java-os/src/test/java/com/baeldung/java9/process/ProcessUnderstandingUnitTest.java

```diff
@@ -16,28 +16,6 @@
 
 class ProcessUnderstandingUnitTest {
 
-    @Test
-    public void givenSourceProgram_whenExecutedFromAnotherProgram_thenSourceProgramOutput3() throws IOException {
```

Comment:
can we add a reason why these tests will not work and disabled them instead of removing?